extensions/amp-form/0.1/amp-form.js

```diff
@@ -556,14 +556,14 @@ export class AmpForm {
       // Validity checking should always occur, novalidate only circumvent
       // reporting and blocking submission on non-valid forms.
       const isValid = checkUserValidityOnSubmission(this.form_);
-      if (this.shouldValidate_ && !isValid) {
+      if (this.shouldValidate_) {
```

Comment:
Oh I see, I wasn't looking at this bigger picture and I didn't see this is only called on submit. I thought this was a more generally useful method. Looks good then 👍